src/views/EmployeePage.vue

```diff
@@ -16,27 +16,112 @@
       <div class="info-block">
         <h4 class="employee-name">
           <md-icon>account_circle</md-icon>
-          {{ `${employee.first_name} ${employee.last_name}` }}
+          <span v-if="!editModeEnabled">
+            {{ `${employee.first_name} ${employee.last_name}` }}
+          </span>
+          <div
+            v-if="editModeEnabled" 
```

Comment:
use v-else

Reply:
throughout the component make it:
**if** editModeEnabled **then** show inputs **else** show without inputs (spans, etc)